test/e2e/libpod_suite_test.go

```diff
@@ -41,6 +41,7 @@ var (
 	redis              = "docker.io/library/redis:alpine"
 	registry           = "docker.io/library/registry:2"
 	infra              = "k8s.gcr.io/pause:3.1"
+	labels             = "quay.io/baude/alpine_labels:latest"
```

Comment:
I know it's just a test, but should there be a "baude" reference?  Can we do something more generic?

Reply:
no, it is a custom image for testing